src/elfpy/utils/price.py

```diff
@@ -87,14 +147,10 @@ def calc_base_asset_reserves(
 
 def calc_liquidity(
     target_liquidity: float,
-    market_price: float,
-    apr: float,
-    time_remaining: StretchedTime,
-    init_share_price: float = 1,
-    share_price: float = 1,
-):
-    """
-    Returns the reserve volumes and total supply
+    target_apr: float,
+    market: Market,
+) -> tuple[float, float]:
+    """Returns the reserve volumes and total supply
 
     The scaling factor ensures token_asset_reserves and base_asset_reserves add
```

Comment:
Please change `token_asset_reserves` and `base_asset_reserves` to `bond_reserves` and `share_reserves`, respectively.